.github/workflows/release-vscode-extension.yml

```
@@ -68,11 +79,15 @@ jobs:
           # Install dependencies
           npm ci
 
-          # Mac first
+          # Mac x86_64 first
           mv ../binary-macos-latest/cargo-concordium executables/cargo-concordium
           chmod +x executables/cargo-concordium
-
           npx vsce package --target darwin-x64 --out ./out/extension-darwin-x64.vsix
+
+          # Then Mac ARM64
+          rm -rf executables/*
+          mv ../binary-macos-latest/cargo-concordium-aarch64 executables/cargo-concordium
```

This can't be right? When specifying multiple paths for `action/upload-artifact` the least common ancestor directory is used as the root ([docs](https://github.com/actions/upload-artifact?tab=readme-ov-file#upload-using-multiple-paths-and-exclusions)). That would mean this

```suggestion
          mv ../binary-macos-latest/aarch64-apple-darwin/cargo-concordium-aarch64 executables/cargo-concordium
```